### PR TITLE
[improve](move-memtbale) enable move memtable in routine load

### PR DESCRIPTION
**Pull request by sollhui**
## Proposed changes

Enable move memtable in routine load.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by HHoflittlefish777** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -523,9 +523,9 @@ public class Config extends ConfigBase {
             "Default pre-commit timeout for stream load job, in seconds."})
     public static int stream_load_default_precommit_timeout_second = 3600; // 3600s
 
-    @ConfField(mutable = true, masterOnly = true, description = {"Stream Load 是否默认打开 memtable 前移",
-            "Whether to enable memtable on sink node by default in stream load"})
-    public static boolean stream_load_default_memtable_on_sink_node = false;
+    @ConfField(mutable = true, masterOnly = true, description = {"导入是否默认打开 memtable 前移",
```
Comment (kaijchen) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:526]: this config does not affect insert into

**Review by kaijchen — CHANGES_REQUESTED** [review]

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by kaijchen — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**